libs/components/src/lib/video-player/video-player.spec.ts

```diff
@@ -0,0 +1,286 @@
+import { axe, elementUpdated, fixture } from '@vivid-nx/shared';
+import { FoundationElementRegistry } from '@microsoft/fast-foundation';
+import { MediaSkipBy } from '../enums';
+import { DEFAULT_PLAYBACK_RATES, VideoPlayer } from './video-player';
+import { videoPlayerDefinition } from './definition';
+import '.';
+
+const COMPONENT_TAG = 'vwc-video-player';
+
+const VIDEO_SRC = 'video.mp4';
+
+jest.mock('video.js', () => {
+	const actualVideoJS = jest.requireActual('video.js');
+
+	return {
+		__esModule: true,
+		default: actualVideoJS
+	};
+});
+
+describe('vwc-video-player', () => {
+	let element: VideoPlayer;
+
+	beforeEach(async () => {
+		element = (await fixture(
+			`<${COMPONENT_TAG}>
+				<source src="${VIDEO_SRC}" type="video/mp4">
+			</${COMPONENT_TAG}>`
+		)) as VideoPlayer;
+	});
+
+	function getBigPlayButton() {
+		return element.shadowRoot?.querySelector('.vjs-big-play-button') as HTMLButtonElement;
+	}
+
+	function getVideoEle() {
+		return element.shadowRoot?.querySelector('video') as HTMLVideoElement;
+	}
+
+	describe('basic', () => {
+		it('should be initialized as a vwc-video-player', async () => {
+			expect(videoPlayerDefinition()).toBeInstanceOf(FoundationElementRegistry);
+			expect(element).toBeInstanceOf(VideoPlayer);
+		});
+
+		it('should be initialise in its default state', async () => {
+			expect(element.src).toBe(undefined);
+			expect(element.poster).toBe(undefined);
+			expect(element.autoplay).toBe(false);
+			expect(element.loop).toBe(false);
+			expect(element.skipBy).toBe(MediaSkipBy.Ten);
+			expect(element.playbackRates).toBe(DEFAULT_PLAYBACK_RATES);
+		});
+
+		it('should show the big play button by removing the vjs-hidden class', async () => {
+			const bigPlayBtn = getBigPlayButton();
+			expect(bigPlayBtn?.classList.contains('vjs-hidden')).toBe(false);
+		});
+
+		it('should remove the redundant lang attribute', () => {
+			expect(element.shadowRoot?.querySelector('[lang]')).toBe(null);
+		});
+	});
+
+	describe('src', () => {
+		it('should show the big play button by removing the vjs-hidden class', async () => {
+			element = (await fixture(
+				`<${COMPONENT_TAG} src="${VIDEO_SRC}"></${COMPONENT_TAG}>`
+			)) as VideoPlayer;
+			await elementUpdated(element);
+			const bigPlayBtn = getBigPlayButton();
+			expect(bigPlayBtn?.classList.contains('vjs-hidden')).toBe(false);
+		});
+
+		it('should allow the src to be updated', async () => {
+			element = (await fixture(
+				`<${COMPONENT_TAG} autoplay>
+					<source src="${VIDEO_SRC}" type="video/mp4">
+				</${COMPONENT_TAG}>`
+			)) as VideoPlayer;
+			await elementUpdated(element);
+			element.src = 'new-src.mp4';
+			await elementUpdated(element);
+			const bigPlayBtn = getBigPlayButton();
+			expect(bigPlayBtn?.classList.contains('vjs-hidden')).toBe(false);
+		});
+	});
+
+	describe('autoplay', () => {
+		it('should set the muted attribute on the video element', async () => {
```

Comment:
This description doesn't seem to be the case anymore. Can you set a more accurate description please?

```suggestion
		it('should set reflect autoplay on the video element', async () => {
```